src/parser/druid/restoration/modules/features/Checklist/Component.js

```diff
@@ -53,22 +53,30 @@ class RestorationDruidChecklist extends React.PureComponent {
             </>
           )}
         >
-          <Requirement 
+          <Requirement
             name={(
               <>
                 <SpellLink id={SPELLS.WILD_GROWTH.id} /> / <SpellLink id={SPELLS.REJUVENATION.id} /> ratio
               </>
             )}
             tooltip="This is your ratio of Wild Growth casts to Rejuvenation casts. If this number is too low, it probably indicates you were missing good opportunities to cast Wild Growth."
             thresholds={thresholds.wildGrowthRatio} />
-          <Requirement 
+          <Requirement
             name={(
               <>
                 Low target <SpellLink id={SPELLS.WILD_GROWTH.id} /> casts
               </>
             )}
             tooltip="This is your percent of Wild Growth casts that hit too few wounded targets. Low target casts happen either by casting it when almost all the raid was full health, or casting it on an isolated target. Remember that Wild Growth can only apply to players within 30 yds of the primary target, so if you use it on a target far away from the rest of the raid your cast will not be effective."
             thresholds={thresholds.wildGrowthPercentBelowRecommendedCasts} />
+          <Requirement
+            name={(
+              <>
+                High initial overhealing <SpellLink id={SPELLS.WILD_GROWTH.id} />
+              </>
+            )}
+            tooltip="This is your percent of Wild Growth casts that has high initial overhealing. Wild Growh does most of it's healing initially and declines over duration. Make sure you are not precasting it before damaging event but after damage occurs."
```

Comment:
Spelling mistake of Wild Growth at the beginning of the second sentence.

Reply:
:+1: